packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -214,6 +219,27 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
   /// The nodes being kept alive despite not being visible.
   final Map<int, RenderBox> _keepAliveBucket = <int, RenderBox>{};
 
+  List<RenderBox> _debugDanglingKeepAlives;
+
+  /// Indicates whether child order verification is enabled
+  ///
+  /// It will perform integrity check every time it is set
```

Comment:
"It" has no antecedent here. Consider something like "Setting this property to true will immediately perform an integrity check".

Then explain what an integrity check is.

Reply:
added comment